src/main/java/nextstep/subway/domain/Section.java

```diff
@@ -0,0 +1,113 @@
+package nextstep.subway.domain;
+
+import nextstep.subway.exception.SectionException;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import static nextstep.subway.exception.SectionExceptionMessage.LESS_THEN_ZERO_DISTANCE;
+
+@Entity
+public class Section extends BaseEntity {
+
+    @Id
+    @Column(name = "section_id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    @JoinColumn(name = "line_id", nullable = false)
+    private Line line;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "up_station_id", nullable = false)
+    private Station upStation;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "down_station_id", nullable = false)
+    private Station downStation;
+    private long distance;
+
+    protected Section() {
```

Comment:
```suggestion
    protected Section() {}
```
[컨벤션](https://naver.github.io/hackday-conventions-java/#permit-concise-empty-block)에 따르면 이렇게 해줄 수도 있을 것 같아요..!